documentation/ntfs/ntfs_fare_extension.md

```diff
@@ -0,0 +1,69 @@
+NTFS - Fare extension
+======================================
+
+# Introduction
+This document describes an extension of the core [NTFS (fr)](./ntfs_fr.md) data format to add fares and tickets information into Navitia.
+The [old fare extension](./ntfs_fare_extension_fr_deprecated.md) is still in use in conjunction with the files described here for backward compatibility matters.
+
+# Data format
+The data format of the files are the same as the core [NTFS ones](./ntfs_fr.md). In short, files are CSV encoded in utf8 and with a comma `,` separator.
+
+# Extension description
+## Added files of this extension
+
+File | Constraint | Comment
+--- | --- | ---
+tickets.txt | Required | Contains tickets declaration.
+ticket_prices.txt | Required | Contains prices of a ticket.
+ticket_uses.txt | Required | Contains where the ticket can be used
+ticket_use_perimeters.txt | Required | Defines the networks and lines of a ticket_use
+ticket_use_restrictions.txt | Optional | Defines a use of a ticket between zones or stop_areas
+
+## Other modifications
+The `object_properties.txt` and `object_codes.txt` files are extended to enable the reference to a ticket_id.
+
+# Added files descriptions
+
+## tickets.txt (Required)
+Field | Type | Constraint | Description
+--- | --- | --- | ---
+ticket_id | Text | Required | Unique ID of the ticket
+ticket_name | Text | Required | Name of the ticket (may be displayed to travelers)
+ticket_comment | Text | Optional | Comment on the ticket that may be displayed to traveler
+
+## ticket_prices.txt (Required)
+Field | Type | Constraint | Description
+--- | --- | --- | ---
+ticket_id | Text | Required | Unique ID of the ticket
+ticket_price | Non-negative Float | Required |
+ticker_currency | Currency Code | Required | An ISO 4217 alphabetical currency code. For the list of current currency, refer to https://en.wikipedia.org/wiki/ISO_4217#Active_codes.
+ticket_validity_start | Date | Required | First day of usability of the ticket (included)
```

Comment:
Do we need some notion of a timezone in this field?

Reply:
I don't think we need some, as the ticket applies to a network (or some lines) that have one.